src/components/InputFields.jsx

```diff
@@ -73,14 +75,15 @@ const InputFields = ({
 
     return (
         <div className="input-fields">
-            {isSettingsPage ? (
+            {isSettingsPage ? ( 
```

Comment:
This condition here indicates that we could split the following 2 parts into 2 different components.